lib/integrations/Exchange/components/AddExchangeIntegration.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import {
+  DialogHeader,
+  DialogProps,
+  Dialog,
+  DialogContent,
+  DialogClose,
+  DialogFooter,
+} from "@components/Dialog";
+import { Form, TextField } from "@components/form/fields";
+import { Alert } from "@components/ui/Alert";
+import Button from "@components/ui/Button";
+
+type Props = {
+  onSubmit: () => void;
+};
+
+export const ADD_EXCHNAGE_INTEGRATION_FORM_TITLE = "addExchange";
```

Comment:
```suggestion
export const ADD_EXCHANGE_INTEGRATION_FORM_TITLE = "addExchange";
```
Exchange is misspelt :)